### DISCUSSION TITLE
如果遇到简悦 · 同步助手无法使用 PDF 等问题请看这里的解决方案

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#1566] [重要通知]
### 问题描述：

RT，包括：**PDF** **Epub** **HTML** 等均无法正常导出

### 解决方案（Mac 用户）

> 请重新在 [下载地址](http://ksria.com/simpread/docs/#/Sync?id=下载) 下载 Mac 版本

###  解决方案（Window 用户）

> 请 **手动设置导出文件夹** 即可解决，如下图：

![image](https://user-images.githubusercontent.com/81074/100043931-67915480-2e49-11eb-960c-100622d8b3d0.png)

PS：因为 Windows 的相同问题还未发布。（发布后我会在这里通知）

**Comment by vinda666** [comment]
Windows 64位系统在手动导出文件夹并配置到网页之后出现了无法打开选项页的情况

**Reply by vinda666** [reply]
并且无法使用 PDF的问题依然存在

**Reply by Kenshin** [reply]
1. 无法打开选项页是因为配置文件有问题，请升级到 2.1 版本，会适度的解决这个问题。
2. （接上条），也可能是类似 https://github.com/Kenshin/simpread/issues/1909 问题导致的
3. PDF 无法导出的问题请看 https://github.com/Kenshin/simpread/discussions/1617 https://github.com/Kenshin/simpread/discussions/1564 的解决方案
